### DISCUSSION TITLE
Does OpenToFu has a CDK like terraform?

### DISCUSSION THREAD

**Discussion by rodrigoscferraz** [opentofu/opentofu#625] [Q&A] [answered]
Hello folks, thanks for this great project! 

Would like to know if OpenToFu already has something like [Terraform CDK](https://developer.hashicorp.com/terraform/cdktf) or if it already is on the roadmap.

Thanks in advance

**Comment by serdardalgic** [comment, marked as answer]
You can follow https://github.com/opentofu/opentofu/issues/601 feature request.

**Reply by rodrigoscferraz** [reply]
Thank you!
